doc/devices/remote.rst

```diff
@@ -4,7 +4,26 @@ The Remote device
 The ``'qiskit.remote'`` device is a generic adapter to use any Qiskit backend as interface
 for a PennyLane device.
 
-This device is useful when retrieving backends from providers with complex search options in
+To access IBM backends, we recommend using `Qiskit Runtime <https://docs.quantum.ibm.com/api/migration-guides/qiskit-runtime-from-ibm-provider>`_.
+
+.. code-block:: python
+
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+    QiskitRuntimeService.save_account(
+        channel="ibm_quantum", 
+        token="<IQP_TOKEN>", 
+        overwrite=True, 
+        default=true
```

Comment:
At the least `True` should be capitalized but in my notebook it rejects the `default` kwarg entirely - maybe I have the wrong version? I get `TypeError: QiskitRuntimeService.save_account() got an unexpected keyword argument 'default'`

Reply:
That... is strange considering I copy pasted this straight off of [Qiskit's documentation](https://docs.quantum.ibm.com/api/migration-guides/qiskit-runtime-from-ibmq-provider) (Not the first time they had a non-working code example). Though... I'm relatively sure I also ran this code myself and it was fine. Let me double check and I'll get back to you.

Reply:
This is probably the safest way to save an account
`QiskitRuntimeService.save_account(channel="ibm_quantum", token="MY_IBM_QUANTUM_TOKEN")`

Seems like the default keyword is not a keyword that actually exists. Well, just don't read that migration guide I guess :)